health/notifications/alarm-notify.sh.in

```diff
@@ -375,6 +375,7 @@ EMAIL_PLAINTEXT_ONLY=
 IRC_NICKNAME=
 IRC_REALNAME=
 IRC_NETWORK=
+IRC_PORT=6667
```

Comment:
Hello @RaitoBezarius ,

Thank you for your contributions,

The IRC values inside this file are modified inside file `health_alarm_notify.conf` present at `health/notifications/health_alarm_notify.conf`, by default, these are the values inside the file:

```conf
# enable/disable sending irc notifications
SEND_IRC="YES"

# if a role's recipients are not configured, a notification will not be sent.
# (empty = do not send a notification for unconfigured roles):
DEFAULT_RECIPIENT_IRC=""

# The irc network to which the recipients belong. It must be the full network.
# e.g. "irc.freenode.net"
IRC_NETWORK=""

# The irc nickname which is required to send the notification. It must not be
# an already registered name as the connection's MODE is defined as a 'guest'.
IRC_NICKNAME=""

# The irc realname which is required in order to make the connection and is an
# extra identifier.
IRC_REALNAME=""
```

Please, can you also add this new variable inside the file?

Reply:
Sure thing!